src/Components/CreateImageWizard/utilities/requestMapper.ts

```diff
@@ -137,6 +143,70 @@ const getLatestRelease = (distribution: Distributions) => {
     : distribution;
 };
 
+function commonRequestToState(
+  request: BlueprintResponse | BlueprintExportResponse
+) {
+  return {
+    details: {
+      blueprintName: request.name,
+      blueprintDescription: request.description,
+    },
+    env: {
+      serverUrl: request.customizations.subscription?.['server-url'] || '',
+      baseUrl: request.customizations.subscription?.['base-url'] || '',
+    },
+    openScap: {
+      profile: request.customizations.openscap
+        ?.profile_id as DistributionProfileItem,
+    },
+    fileSystem: request.customizations.filesystem
+      ? {
+          mode: 'manual' as FileSystemConfigurationType,
+          partitions: request.customizations.filesystem.map((fs) =>
+            convertFilesystemToPartition(fs)
+          ),
+        }
+      : {
+          mode: 'automatic' as FileSystemConfigurationType,
+          partitions: [],
+        },
+    firstBoot: {
+      script: getFirstBootScript(request.customizations.files),
+    },
+    distribution: getLatestRelease(request.distribution),
+    repositories: {
+      customRepositories: request.customizations.custom_repositories || [],
+      payloadRepositories: request.customizations.payload_repositories || [],
+      recommendedRepositories: [],
+    },
+    registration: {
```

Comment:
Given we [filter out the registration](https://github.com/osbuild/image-builder/blob/767cb9ce2496146e193d5c58ebbafe68956d5ab3/internal/v1/handler_blueprints.go#L254), it should probably be ommited for export, right (meaning not be in the common section)?